docs/configuration.md

```diff
@@ -160,6 +166,34 @@ info: CarbonAware.DataSources.Json.JsonDataSource[0]
     Reading Json data from /app/data-sources/json/mycustomfile.json
 ```
 
+### ElectricityMaps Configuration
+
+If using the ElectricityMaps data source, ElectricityMaps configuration is required.
+
+```json
```

Comment:
Here is a spot we may want to note the 2 different configurations (for token users vs. free trial users)